include/megaapi.h

```diff
@@ -1376,7 +1488,7 @@ class MegaStringList
      * @param i Position of the string that we want to get for the list
      * @return string at the position i in the list
      */
-    virtual const char* get(int i);
+    virtual const char* get(int);
```

Comment:
It's needed to add the name of the parameter again to not break the documentation